src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
+
+    public RightTriangle(double firstLeg,double secondLeg, Color color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        double result = (firstLeg * secondLeg) / 2;
+        return Math.sqrt(result);
```

Comment:
there's no `sqrt` in area's formula
```suggestion
        return firstLeg * secondLeg / 2;
```